src/ol/observable.js

```diff
@@ -62,6 +62,18 @@ ol.Observable.prototype.changed = function() {
 
 
 /**
+ * Dispatches an event (or event like object) and calls all listeners
+ * listening for events of this type. The type of the event is decided by the
+ * type property on the event object.
+ *
+ * @param {goog.events.EventLike} event Event object.
+ * @function
+ * @api
+ */
+ol.Observable.prototype.dispatchEvent;
```

Comment:
it's not `undefined`: `ol.Observable.prototype.dispatchEvent;` has no effect at all. It only tells the export system to expose the function.

The same system is used elsewhere in ol3, for example: https://github.com/openlayers/ol3/blob/master/src/ol/format/geojsonformat.js#L374-L384